src/common/components/hosts/MassDeleteHostModal.tsx

```diff
@@ -20,15 +20,18 @@ type MassDeleteHostModalProps = {
   hosts: Host[];
   // eslint-disable-next-line
   onDelete: (host: Host) => Promise<any>;
+  reloadCluster: VoidFunction;
```

Comment:
This property should be optional given that `InfraEnvAgentTable` in CIM uses this component.

Same in `MassChangeHostnameModal`